src/expression.ts

```diff
@@ -98,20 +101,27 @@ export class ElementAccessExpr extends BaseNode<"ElementAccessExpr"> {
   }
 }
 
+export const isArgumentExpr = typeGuard("ArgumentExpr");
+
+export class ArgumentExpr extends BaseNode<"ArgumentExpr"> {
```

Comment:
Duplicate change in two PRs?

Reply:
Yeah, this is on top of the other PR #47. I could set the other PR branch as the parent of this PR. Not sure the right way to handle concurrent PRs.